fastparquet/encoding.py

```diff
@@ -4,10 +4,18 @@
 from __future__ import division
 from __future__ import print_function
 
+
 import array
-import numba
+
 import numpy as np
 
+from packaging import version
+import numba
+if version.parse(numba.__version__) < version.parse("0.49"):
```

Comment:
Most people seem to use LooseVersion for this kind of task, presumably because it's less strict around "rc", "beta", "+git" labels and such.

Reply:
would you prefer to use LooseVersion? 

I'm agnostic, but this answer suggests `packaging` is compliant with the latest PEP https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

I just tested `packaging.version` with the latest numba dev version: `'0.50.0dev0+64.gcef0946e9'` and it seems to work well.

```python

version.parse("0.49") < version.parse(numba.__version__)
#true
```

Your call, I'm ok either way. Alternatively, I could remove the `if` entirely, but that would kill support for python < 3.6, and possibly many pandas and numpy versions.

Reply:
ok fine, leave it. 
However, a version which is installing v0.49 is still generating a warning https://travis-ci.org/github/dask/fastparquet/jobs/679174878#L608 and I have no idea what's up with the py35 install.

Reply:
I fixed the warning and had a look at the py35 build: the issue comes from building the wheel for llvmlite. the gcc version and make file might not be compatible.